src/index.ts

```diff
@@ -622,7 +623,19 @@ const initialize = (apiKey: string, target: Target, options?: Options): Result =
     eventSourceWithMiddleware = addMiddlewareToEventSource(middleware)
   }
 
-  return { on, off, variation, close, setEvaluations, registerAPIRequestMiddleware }
+  const refreshCache = (): boolean => {
```

Comment:
I don't think we should call this `cache`. The only thing that's exposed to the SDK user that uses `cache` is completely unrelated and this will cause confusion. Maybe `refeshEvaluations`

Reply:
can be easily changed